store/storetest/status_store.go

```diff
@@ -68,11 +68,10 @@ func testStatusStore(t *testing.T, ss store.Store) {
 		t.Fatal(err)
 	}
 
-	if result := <-ss.Status().Get(status.UserId); result.Err != nil {
-		t.Fatal(result.Err)
+	if statusParameter, err := ss.Status().Get(status3.UserId); err != nil {
```

Comment:
You are changing here the variable used to get the status from `status` to `status3`